apps/state/config/config.exs

```diff
@@ -218,7 +218,7 @@ config :state, :stops_on_route,
     # Kingston Line shuttles to/from South Weymouth
     "Shuttle-BraintreeSouthWeymouth-0-" => true,
     # Providence trains stopping at Forest Hills
-    "CR-Providence-d01bc229-0" => true,
```

Comment:
See PR description for more details but this route pattern 404s at the moment:
https://api-dev.mbtace.com/route_patterns/CR-Providence-d01bc229-0